README.md

```diff
@@ -20,7 +20,8 @@ by boosting any typical **Linux text-processing utils** such as `grep`, `sort`,
 To start using **up**, redirect any text-emitting command (or pipeline) into it
 — for example:
 
-    $ lshw |& ./up
+    wget https://github.com/akavel/up/releases/download/v0.3/up
```

Comment:
```suggestion
    wget https://github.com/akavel/up/releases/download/v0.3/up
    chmod 700 up
```

Reply:
You can download always the latest Version: wget https://github.com/akavel/up/releases/latest/download/up